code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -131,6 +159,17 @@
 
 /mob/living/simple_animal/hostile/giant_spider/Initialize(mapload, atom/parent)
 	get_light_and_color(parent)
+	add_language(LANGUAGE_GREIMORIAN)
+	add_language(LANGUAGE_GREIMORIAN_HIVEMIND)
+	. = ..()
+
+/mob/living/simple_animal/hostile/giant_spider/nurse/servant/Initialize()
+	add_verb(src, /mob/living/proc/ventcrawl)
+	var/number = rand(1000,9999)
+	name = initial(name) + " ([number])"
+	real_name = name
+	if(playable && !ckey && !client)
+		SSghostroles.add_spawn_atom("servant", src)
 	. = ..()
```

Comment:
Call to parent in init should be done before anything else, as it sets the required flags and prepares the atom with the parent needs, while ensuring they aren't overriding the child, eg. if `/mob/living/simple_animal/hostile/giant_spider/nurse` were to set the `real_name` to something else, this would override what you set it here as

There are few instances were it's permissible (due to shitcode) to call parent later in the proc, this most likely isn't one of those

Reply:
handled